class7-week3/actions/markAsDone.js

```diff
@@ -0,0 +1,24 @@
+const Todo = require('../models/todo')
+const deserializeTodo = require('../util/deserializeTodo')
+
+module.exports = function update(request, response) {
+    
+  const id = request.params.id
+  console.log(request.params)
+  const todo = deserializeTodo(request, response)
```

Comment:
Same comments apply to markAsNotDone.js

You should comment out lines 8 and 9. Those are only useful when you're extracting a todo from the body of a POST/PUT request. We aren't doing that here, so there's nothing to deserialize.